pyroomacoustics/room.py

```diff
@@ -523,9 +523,19 @@ def callback_mix(premix, snr=0, sir=0, ref_mic=0, n_src=None, n_tgt=None):
 def wall_factory(corners, absorption, scattering, name=""):
     """ Call the correct method according to wall dimension """
     if corners.shape[0] == 3:
-        return Wall(corners, absorption, scattering, name,)
+        return Wall(
+            corners,
+            absorption,
+            scattering,
+            name,
+        )
```

Comment:
Regarding this formatting edits, I guess there's a bunch of places that could use such fixes. Should we run something like [black](https://github.com/psf/black) across the whole package?